### PR TITLE
TRAN-87: Added searcher and table to the Customer master page

### PR DESCRIPTION
**Pull request by KarolinaKopacz**
**Add a table with Customer data and filtering**

**Implemented solution detail:**

TRAN-87

### CODE REVIEW

File: src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      Header: format('customer_master.phone.label'),
+      accessor: 'phone',
+    },
+    {
+      Header: format('customer_master.gps.label'),
+      accessor: 'latitude_longitude',
+    },
+  ], [format]);
+
+  // const dispatch = useDispatch();
```
Comment (sniedzielski) [inline src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx]: can you remove this?

**Review by sniedzielski — CHANGES_REQUESTED** [review]
@KarolinaKopacz Please fix one line. Also I wonder if we should rename CustomerMaster into 'Customer'.

File: src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx (hunk 1 above)
Comment (KarolinaKopacz) [reply src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx]: Of course!

Comment (KarolinaKopacz) [reply src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx]: @sniedzielski ready!

**Review by sniedzielski — APPROVED** [review]

**Pull request merged**